index.html

```diff
@@ -10467,6 +10484,44 @@ <h3>Relationship Attributes</h3>
 		<h2>Definitions of States and Properties (all aria-* attributes)</h2>
 		<p>Below is an alphabetical list of <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> <a>states</a> and <a>properties</a> to be used by rich internet application authors. A detailed definition of each <abbr title="Accessible Rich Internet Applications">WAI-ARIA</abbr> state and <a>property</a> follows this compact list.</p>
 		<p id="index_state_prop" class="placeholder">Placeholder for index of states and properties</p>
+		<div class="property" id="aria-actions">
+			<pdef>aria-actions</pdef>
+			<div class="property-description">
+				<p><a>Identifies</a> a related element or elements whose primary activation (<code>click</code> event) will trigger behaviors or operations relevant to the current element.</p>
+				<p>The <code>aria-actions</code> attribute provides an element to other interactive elements which trigger actions on the current element. For example, in a web mail application, the currently selected email message may reference action elements such as the <em>message-specific</em> Reply, Forward, and Delete buttons. Actions are triggered by mainstream pointer events (click, tap) or by simulated pointer events from assistive technology.</p>
+				<ul>
+					<li>Authors MUST ensure that related action elements have a label.</li>
+					<li>Authors MUST ensure that related action elements accept a <code>click</code> event handler or default activation behavior.</li>
+					<li>Authors MUST ensure that related action events are not limited to a modality-specific activation such as the <code>keyup</code> or <code>touchend</code> event handlers.</li>
+					<li>Authors SHOULD ensure that related actions elements have subclass role of <rref>command</rref> such as <rref>button</rref> or <rref>link</rref>. [Todo: What about the "activatable" <rref>input</rref> subclass roles like <rref>checkbox</rref> and <rref>switch</rref><rref>? I think UAs could make those work by combining label and status: action name could be "uncheck 'subscribe to marketing emails'". -jc]</li>
```

Comment:
Perhaps we define some subclass of input that are click-actionable inputs? (e.g. checkbox, radio, switch but not spinbutton, menu, etc), then this could be "Authors SHOULD ensure that related actions elements have subclass role ofcommand or bikeshed-actionable-input…"

Reply:
I think we need to talk about this one, since we were initially leaning towards limiting it to widget roles, rather than command subclasses + some input subclasses.

There was one use case in particular that I think would be good to look at: a menu item that has a nested text input or `<select>`, e.g. "Snooze reminder for [30 minutes]" where "30 minutes" is a dropdown of time options.

Reply:
The general feeling within the Apple team is that VoiceOver or Switch Control users would not expect an action to cause a simple focus change... So aria-actions pointing to an HTML `<select>` would be odd... the synth click brings focus to the submenu, which pulls the user away from the original context, but doesn't actually perform an action.

Part of the benefit and usability of the native actions API is that it leaves the user's AT focus on the original item for most actions, rather than forcing another navigation stop for each secondary action... 

![Gmail messages table with actions: archive, delete, etc on the first row](https://user-images.githubusercontent.com/75867/151451855-071b38ed-bb8b-4616-9bc7-176b4ff6d527.png)

Using Gmail as the same example from multiple related issues, a user moves through a preview list e.e., "Sender: Subject X"... The next move is either to "Sender: Subject Y" or performing some action on the current Message X. In Apple's implementation of a similar native UI, there are NOT multiple navigation stops for each "_View_ Sender:Subject X," "_Select_ Sender:Subject X," "_Archive_ Sender:Subject X," "_Delete_ Sender:Subject X," "_Reply to_ Sender:Subject X," "_Forward_ Sender:Subject X," etc., etc... So a web action that just only caused focus change could be problematic to implement for Apple's AT, not from a technical perspective, but a usability one. It could disorient the user without actually performing any action.

That said, the operation that is performed by the action could have the resulting effect of a focus change. Again in the Gmail messages example above:
1. the Archive action would archive self, so focus would need to move to the next logical message,
2. the Reply action would spawn a reply and focus the logical field to edit.
3. etc.

But those are the _result_ of the action that changes the focus, not the action itself.

In the same example, the "Select" or "Favorite/Star" actions ideally would be performed without any focus change. When on Message X, triggering the "Select" action would change the visible checkbox select (with a synthesized click) but ideally would leave focus where it is. "I'm still on the message, but now it's selected… Or starred." 

![Gmail messages table with actions: select, favorite, etc](https://user-images.githubusercontent.com/75867/215601617-dee4744f-ef54-439a-a7cc-73d0b1a2665f.png)

Reply:
I also acknowledge that web authors are going to try to point actions at unexpected things… Like the text field or the select examples... We could call that a validation error, or but many others won't see that, so what should implementations do? 

IMO, AT and UAs should probably still synthesize the click, regardless of the role and expected behavior of the related action element.

Reply:
Even if it's weird in most cases, some web author will figure out how to use that pattern in a good use case. If the implementation actively prevents it, that'd be prioritizing Spec Authors or UA devs over Web authors or Users.

Reply:
(Hence the Authors SHOULD, not Authors MUST)

Reply:
Removing that requirement because I think this is already covered sufficiently by the related "elements accept a click event handler" required (2 lines higher)
```suggestion
```

Reply:
Pulling this one b/c no objections.